indico/web/client/js/react/components/index.js

```diff
@@ -14,6 +14,7 @@ export {
   TrackACLField,
 } from './principals';
 export {default as ConfirmButton} from './ConfirmButton';
+export {default as EventImageSelectField} from './EventImageSelectField';
```

Comment:
I don't think this should pollute the global components. Since it's only used for receipt template stuff for now, let's keep it there.